features/dump_offline_inventories.lua

```diff
@@ -11,13 +11,30 @@ local Config = require 'config'
 local set_timeout_in_ticks = Task.set_timeout_in_ticks
 local config = Config.dump_offline_inventories
 
+local ignored_items_set = {}
+for _ , k in pairs(config.ignored_items) do
```

Comment:
```suggestion
for _ , k in pairs(config.ignored_items or {}) do
```
There are a lot maps that directory set `dump_offline_inventories` with a new table that are missing this new field, search for `dump_offline_inventories =`. Another option would be to go to all those sites and add an empty table.

Reply:
I will add that.